decidim-forms/app/views/decidim/forms/questionnaires/answers/_long_answer.html.erb

```diff
@@ -1 +1 @@
-<%= answer_form.text_area :body, label: false, id: field_id, rows: 10, disabled: disabled, maxlength: maxlength %>
+<%= answer_form.text_area :body, label: false, id: field_id, rows: 10, disabled: disabled, maxlength: maxlength, class: "w-full" %>
```

Comment:
With the default color choices these text questions (both short and long) look like disabled inputs (grey) especially on mobile screens:
![Long answer disabled](https://user-images.githubusercontent.com/864340/183114286-f54c5861-fcbb-4cbe-8bf7-de7f8e3d0fec.png)

Reply:
Slight color change